pkg/cli/admin/release/extract_tools.go

```diff
@@ -276,6 +276,17 @@ func (o *ExtractOptions) extractCommand(command string) error {
 			InjectReleaseVersion: true,
 			ArchiveFormat:        "openshift-client-linux-amd64-rhel9-%s.tar.gz",
 		},
+		{
+			OS:      "linux",
+			Arch:    "amd64",
+			Command: "oc.rhel8",
+			Mapping: extract.Mapping{Image: "cli-artifacts", From: "usr/share/openshift/linux_amd64/oc.rhel8"},
```

Comment:
How's the symlink processing implemented here? Will the extracted binary be a symlink or the symlink target?

Reply:
Will the final binary be oc or oc.rhel8? I prefer oc only.

Reply:
Thanks a lot for review.

> How's the symlink processing implemented here? Will the extracted binary be a symlink or the symlink target?

For which symlink are you asking oc -> kubectl or oc -> oc.rhel8?. If it is for oc -> oc.rhel8, it is the same principle of https://github.com/openshift/oc/blob/f1cac6d7fa098afbc170d3bae5bdce6b5bbecd3d/images/cli-artifacts/Dockerfile.rhel#L18 and https://github.com/openshift/oc/blob/f1cac6d7fa098afbc170d3bae5bdce6b5bbecd3d/pkg/cli/admin/release/extract_tools.go#L246-L256. Extracted binary is a real binary not a symlink.

> Will the final binary be oc or oc.rhel8? I prefer oc only.

Good point and it would be nicer, I can change them to `oc`.

Reply:
@ingvagabund I've added new optional TargetCommandName field to rename extracted binaries.